xml/System.Text.RegularExpressions/Regex.xml

```diff
@@ -348,6 +352,10 @@
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
+
+> [!WARNING]
+> When using <xref:System.Text.RegularExpressions> to process untrusted input, pass a [timeout](/dotnet/standard/base-types/best-practices#use-time-out-values). A malicious user can provide input to `RegularExpressions`, causing a [Denial-of-Service attack](https://www.cisa.gov/news-events/news/understanding-denial-service-attacks), timeout specifies how long a pattern-matching method should try to find a match before it times out.
```

Comment:
```suggestion
> When using <xref:System.Text.RegularExpressions> to process untrusted input, pass a [time-out value](/dotnet/standard/base-types/best-practices-regex#use-time-out-values) to prevent malicious users from causing a [denial-of-service attack](https://www.cisa.gov/news-events/news/understanding-denial-service-attacks). A time-out value specifies how long a pattern-matching method should try to find a match before it times out.
```